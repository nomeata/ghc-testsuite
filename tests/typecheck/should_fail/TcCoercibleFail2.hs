import GHC.Types (Coercible)

instance Coercible () ()

main = return ()
