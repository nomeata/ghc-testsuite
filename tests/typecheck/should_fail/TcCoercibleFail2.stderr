
TcCoercibleFail2.hs:3:10:
    Manual instances forbidden for abstact class Coercible
    In the instance declaration for ‛Coercible () ()’
